themes/bootstrap3/templates/layout/layout.phtml

```diff
@@ -130,10 +130,6 @@
       }
 
       $this->headScript()->appendScript(implode("\n", $appendScripts));
-
-      if ($this->config()->ajaxCoversEnabled()) {
```

Comment:
No, this only decides if the covers.js is loaded or not. Now, we also want to include the file event if we don't to use ajax for the covers, because the unavailable images get hidden with js.

If the covers should be loaded via ajax is decided by adding the "ajaxcover" class, e.g. like here https://github.com/vufind-org/vufind/blob/1390c2efd452bf2f934bd6a15d38867a515798a1/themes/bootstrap5/templates/RecordDriver/DefaultRecord/cover.phtml#L19

Reply:
Thanks for the clarification!